app/placeholder/placeholder-deploy.sh

```diff
@@ -35,6 +50,10 @@ if [[ -n $UPDATED ]]; then
     cat firebase.json
 fi
 
+# Finally, deploy the application
 echo "Deploying placeholder to Firebase..."
-
 firebase deploy --project "$PROJECT_ID" --only hosting
+
+# Setup for greater chances of success by explicitly purging cache
+echo "Purging firebase cache"
+curl -X PURGE "${FIREBASE_URL}/"
```

Comment:
Did we notice a need for cache clear on top of the placeholder?

Reply:
it prevents issues where the deploy may not have caused a purge, and the site may have been pre-loaded elsewhere.